client/packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -92,181 +87,47 @@ const DetailViewInner = () => {
-          if ('subRows' in row)
-            return (
-              ArrayUtils.getUnitQuantity(row.subRows ?? []) *
-              (row.item.doses ?? 1)
-            );
-
-          return row.packSize * row.numberOfPacks * (row.item.doses ?? 1);
-        },
-      },
-      {
-        id: 'unitSellPrice',
-        header: t('label.unit-sell-price'),
-        columnType: ColumnType.Currency,
-        defaultHideOnMobile: true,
-        accessorFn: rowData => {
-          if ('subRows' in rowData) {
-            return ArrayUtils.getAveragePrice(
-              rowData.subRows ?? [],
-              'sellPricePerPack'
-            );
-          } else {
-            if (isDefaultPlaceholderRow(rowData)) return undefined;
-            return (rowData.sellPricePerPack ?? 0) / rowData.packSize;
-          }
-        },
-      },
-      {
-        id: 'total',
-        header: t('label.total'),
-        columnType: ColumnType.Currency,
-        defaultHideOnMobile: true,
-        accessorFn: rowData => {
-          if ('subRows' in rowData) {
-            return Object.values(rowData.subRows ?? []).reduce(
-              (sum, batch) =>
-                sum + batch.sellPricePerPack * batch.numberOfPacks,
-              0
-            );
-          } else {
-            if (isDefaultPlaceholderRow(rowData)) return '';
-
-            const x = rowData.sellPricePerPack * rowData.numberOfPacks;
-            return x;
-          }
-        },
-      },
-    ];
+  const columns = useOutboundColumns();
 
-    return cols;
-  }, [manageVvmStatusForStock]);
-
-  const { table, selectedRows, resetRowSelection } =
+  const { table, selectedRows } =
     useNonPaginatedMaterialTable<StockOutLineFragment>({
       tableId: 'outbound-shipment-detail-view',
-      columns: mrtColumns,
+      columns,
       data: rows ?? [],
-      onRowClick: onRowClick ? row => onRowClick(row) : () => {},
+      groupByField: isGrouped ? 'itemName' : undefined,
       isLoading: false,
+      initialSort: { key: 'itemName', dir: 'asc' },
+      onRowClick: !isDisabled ? onRowClick : undefined,
       getIsPlaceholderRow: row =>
         row.type === InvoiceLineNodeType.UnallocatedStock ||
         row.numberOfPacks === 0,
-      groupByField: isGrouped ? 'itemName' : undefined,
+      renderEmptyRowsFallback: () => (
+        <NothingHere
+          body={t('error.no-outbound-items')}
+          onCreate={isDisabled ? undefined : () => onAddItem()}
+          buttonText={t('button.add-item')}
+        />
+      ),
     });
 
+  // Table manages the sorting state
+  // This needs to be passed to the edit modal, so based on latest sort order
+  // it can determine which item to load when user clicks `next`
+  const getSortedItems = useCallback(
+    () =>
+      table.getSortedRowModel().rows.reduce<ItemRowFragment[]>((acc, row) => {
+        const item = row.original.item;
+        if (!acc.find(i => i.id === item.id)) acc.push(item);
+        return acc;
```

Comment:
returns array of unique items, in the current sort order of the table.

potentially this should be moved into some kind of helper to be reusable, but let's cross that bridge when we get there, not sure yet whether we might want/need some variation in this implementation 😃